pyccl/tests/test_concentration.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pytest
+import pyccl as ccl
+
+
+COSMO = ccl.Cosmology(
+    Omega_c=0.27, Omega_b=0.045, h=0.67, sigma8=0.8, n_s=0.96,
+    transfer_function='bbks', matter_power_spectrum='linear')
+PKA = ccl.Pk2D(lambda k, a: np.log(a/k), cosmo=COSMO)
+CONCS = [ccl.halos.ConcentrationDiemer15,
+         ccl.halos.ConcentrationBhattacharya13,
+         ccl.halos.ConcentrationPrada12,
+         ccl.halos.ConcentrationKlypin11,
+         ccl.halos.ConcentrationDuffy08]
+MS = [1E13, [1E12, 1E15], np.array([1E12, 1E15])]
+# None of our concentration-mass relations
+# are defined for FoF halos.
+MDEF = ccl.halos.MassDef('fof', 'matter')
+
+
+@pytest.mark.parametrize('cM_class', CONCS)
+def test_cM_subclasses_smoke(cM_class):
+    cM = cM_class()
+    for m in MS:
+        c = cM.get_concentration(COSMO, m, 0.9)
+        assert np.all(np.isfinite(c))
+        assert np.shape(c) == np.shape(m)
+
+
+def test_cM_duffy_extra():
```

Comment:
What is this test testing? Maybe a better name?

Reply:
It mostly tests that you can use this parametrization with other input parameters. Changed to `duffy_smoke`